index.js

```diff
@@ -548,6 +549,20 @@ class Squeakquel extends Datastore {
             findParams.order = [[sortKey, 'DESC']];
         }
 
+        if (config.aggregationField) {
+            if (!findParams.attributes) {
+                findParams.attributes = [];
+            }
+
+            findParams.attributes.push(config.aggregationField);
+            findParams.attributes.push([
+                Sequelize.fn('COUNT', Sequelize.col(config.aggregationField)),
```

Comment:
@InderH can you update the sql generated as part of this fn ?

Reply:
SELECT `templateId`, COUNT(`templateId`) AS `count` FROM `jobs` AS `jobs` WHERE `jobs`.`templateId` IN ('12', '11', '10', '9', '8', '7', '3', '2', '1') GROUP BY `templateId` ORDER BY `jobs`.`id` DESC;


SELECT `templateId`, COUNT(`templateId`) AS `count` FROM `builds` AS `builds` WHERE `builds`.`templateId` IN ('12', '11', '10', '9', '8', '7', '3', '2', '1') GROUP BY `templateId` ORDER BY `builds`.`createTime` DESC;